### ISSUE TITLE
Accessibility support in the PDF.js web viewer (web/viewer.html)

### ISSUE THREAD

**Issue by Karthikvenkat86** [rkusa/pdfjs#328]
I’m assessing accessibility support in the PDF.js web viewer (web/viewer.html). Currently, all text—including headers—is rendered as plain <span> elements, without semantic roles like **role="heading"** or aria-level. Are there UI settings I’m missing, or is this a known limitation/bug in the viewer?

As of now, its generates a <span role="application">Header</span> where screen reader cannot identify this as a header.

<img width="1923" height="948" alt="Image" src="https://github.com/user-attachments/assets/52333ce1-72f5-4d32-8547-fd4fd7dd8d42" />

**Comment by rkusa** [comment]
You probably meant to open an issue in this repo: https://github.com/mozilla/pdf.js
